test/test_cftime.py

```diff
@@ -1,4 +1,5 @@
 from __future__ import print_function
+from calendar import c
```

Comment:
Is this needed?

Reply:
No, that looks like a "helpful" addition from vscode auto-import... good spot thanks! will remove